frontend/app/page.js

```diff
@@ -9,89 +10,155 @@ async function getData() {
   return res.json();
 }
 
+// Main component
 export default async function Home() {
   const data = await getData();
 
   return (
     <div>
-      <h1 className='display-1'>PikoWeatherer</h1>
+      <h1 className='display-1' style={{ textAlign: "center", padding: "auto", margin: "auto", marginTop: "1%" }}>PikoWeatherer</h1>
```

Comment:
```suggestion
      <h1 className='display-1 mt-1' style={{ textAlign: "center", padding: "auto", margin: "auto"}}>PikoWeatherer</h1>
```

use bootstrap-provided classes (e.g., `mt-x`, `mb-x`) for margins
https://getbootstrap.com/docs/5.3/utilities/spacing/

Reply:
sir, yessir! o7

Reply:
nice!